internal/endpoints.go

```diff
@@ -323,6 +329,80 @@ func (a *Api) setPlayDateAttendence(c *gin.Context) {
 	c.HTML(http.StatusOK, "partials/players-table.html", state)
 }
 
+func (a *Api) setPlayDateAttendenceFromDisc(r *discordgo.MessageReaction) {
+	if r.UserID == "1252426978313633812" {
+		log.Info().Msg("Reaction created by bot")
+		return
+	}
+
+	log.Info().Any("Reaction", r).Msg("Setting player attendance")
+	discId := r.UserID
+	react := r.Emoji.Name
+	msg, err := a.dg.ChannelMessage(Config.DiscordChannelID, r.MessageID)
+	if err != nil {
+		log.Err(err).Msg("failed to get reaction message")
+		a.dg.ChannelMessageSend(Config.DiscordChannelID, "Couldn't update attendance, sorry.")
+		return
+	}
+	if msg.Author.ID != "1252426978313633812" {
+		log.Info().Msg("Not a bot message")
+		return
+	}
+
+	attendance := AttendanceFrom(react) // parse input attendence action to internal enum
+	msgSplit := strings.Split(msg.Content, "/")
+	if len(msgSplit) <= 1 {
+		log.Info().Msg("Not a playdate")
+		return
+	}
+	pId, err := strconv.Atoi(msgSplit[len(msgSplit)-1])
+	if err != nil {
+		log.Err(err).Msg("failed to parse given playdate id")
+		a.dg.ChannelMessageSend(Config.DiscordChannelID, "Failed to get playdate from message")
+		return
+	}
+
+	playdate := &PlayDate{ID: pId}
+	err = a.db.NewSelect().Model(playdate).WherePK().
+		Where("date <= ?", time.Now().Format("2006-01-02T15:04")).
+		Where("status = ?", PlayDateStatusPending).Scan(a.ctx)
+	if err != nil {
+		log.Err(err).Int("playdateID", pId).Msg("failed to find playdate or it already happened")
+		return
+	}
```

Comment:
what if you removed the where statements from the sql query and we did those checks in go. This would then allow us to differentiate between not finding the playdate and someone trolling